src/main/resources/application.yaml

```diff
@@ -48,7 +48,7 @@ spring:
 featureFlag:
   docmosisEnabled: ${IA_DOCMOSIS_ENABLED:false}
   isEmStitchingEnabled: ${IA_EM_STITCHING_ENABLED:false}
-  isSubmitHearingRequirementsEnabled: ${IA_SUBMIT_HEARING_REQUIREMENTS_ENABLED:false}
+  isSubmitHearingRequirementsEnabled: ${IA_SUBMIT_HEARING_REQUIREMENTS_ENABLED:true}
```

Comment:
flag should be `false`, we can change it later I think

Reply:
Answering in the absence of Tarun. He mentioned it would anyway have been overwritten in env specific configs and to aid easy development he made it true. I will leave it as it is for now.